jdaviz/configs/specviz2d/helper.py

```diff
@@ -59,6 +59,9 @@ def _update_spec2d_x_axis(self, change):
                 setattr(scales['x'], name, idx)
 
     def _update_spec1d_x_axis(self, change):
+        if self.app.get_viewer('spectrum-viewer').state.reference_data is None:
```

Comment:
@havok2063 , some load order must have changed. This code was called before `self.app.get_viewer('spectrum-viewer').state.reference_data` was set, hence the crashing.